blockchain/thresholdstate.go

```diff
@@ -0,0 +1,322 @@
+// Copyright (c) 2016 The btcsuite developers
+// Use of this source code is governed by an ISC
+// license that can be found in the LICENSE file.
+
+package blockchain
```

Comment:
This might be something reasonable to consider for the future, but given the current code is already complete, reviewed, and tested, I don´t think it would make sense to do it for this PR.